service/ansible/roles/atmo-setup-user/tasks/main.yml

```diff
@@ -1,12 +1,14 @@
+- name: add core services to sudoers
+  lineinfile:
+    dest=/etc/sudoers
+    backup=yes
+    line="{{ item }}" 
+  with_items: 
+    - '%core-services ALL=(ALL) ALL'  
+
```

Comment:
Would be better if this was a variable that could be passed in, to avoid installation-specific code into the codebase.